scripts/exglobal_atmos_pmgr.sh

```diff
@@ -71,7 +73,7 @@ do
 
   sleep 10
   icnt=$((icnt + 1))
-  if [ $icnt -ge 1080 ]
+  if [ $icnt -ge 1001 ]
```

Comment:
@lgannoaa May I ask the reason of changing 1080 into 1001? I recalled the 1080 was added based on NCO GFS V16 testing. @KateFriedman-NOAA may chime in.

Reply:
@lgannoaa Got it. Thanks for clarifying.